api/api.js

```diff
@@ -1,7 +1,9 @@
+import logger from 'lib/infrastructure/logging/logger';
 import { HttpStatusError } from './HttpStatusError';
 
 async function request(path, { token, ...options }) {
   const url = `${process.env.NEXT_PUBLIC_API_URL}${path}`;
+  logger.info(`Fetching ${url}`, options?.body ? options.body : null);
```

Comment:
`options?.body` should produce the same output as `options?.body ? options.body : null` I think?

Reply:
ha true